tests/integration_tests/without_db/udfs/test_filling_mask_udf.py

```diff
@@ -85,6 +85,13 @@ def test_filling_mask_udf(
     sequence_classifier.run(ctx)
 
     result_df = ctx.get_emitted()[0][0]
-    assert result_df.shape == (n_rows * top_k, 7) and \
-           result_df['score'].dtypes == 'float' and \
-           list(result_df.columns) == columns[1:] + ['filled_text', 'score']
+    new_columns = ['filled_text', 'score', 'rank']
+    assert result_df.shape == (n_rows * top_k, len(columns)+len(new_columns)-1) \
+           and result_df.shape[1] == len(columns) + len(new_columns) - 1 \
+           and list(result_df.columns) == columns[1:] + new_columns \
+           and result_df['score'].dtypes == 'float' \
+           and result_df['rank'].dtypes == 'int' \
+           and all([result_df[row*top_k: top_k + row*top_k]
+                   .sort_values(by='score', ascending=False)['rank']
+                   .is_monotonic for row in range(n_rows)])
```

Comment:
The assertions are very hard to ready from my point of view, what about doing something like this?

```python
    has_valid_shape = result_df.shape == (n_rows * top_k, len(columns)+len(new_columns)-1)
    score_is_typed_as_float = result_df['score'].dtypes == 'float'
    rank_is_typed_as_int = and result_df['rank'].dtypes == 'int'
    ...

    assert all((
        has_valid_shape, 
        score_is_typed_as_float,
        rank_is_typed_as_int,
    ))

```

Reply:
thanks, agree, this way is better for readability